### PR TITLE
Documantation Automation

### PR DESCRIPTION
**Pull request by Itay4**
Fixes https://github.com/demisto/server/issues/9850

### CODE REVIEW

**Comment by dorsha** [comment]
Please share output screenshots

**Comment by Itay4** [comment]
Output screenshot:

screen shot 2017-11-26 at 13 08 06

**Comment by Spivaka** [comment]
I would also add parameters to the "configure the following settings" part

File: Playbooks/playbook-DocumantationAutomation.yml [hunk 1]
```diff
@@ -0,0 +1,67 @@
+id: DocumantationAutomation Playbook
```
Comment (anara123) [inline Playbooks/playbook-DocumantationAutomation.yml]: You have typo Documantation => Documentation

File: Scripts/script-DocumentationAutomation.yml [hunk 2]
```diff
@@ -0,0 +1,128 @@
+commonfields:
+  id: DocumentationAutomation
+  version: -1
+name: DocumentationAutomation
+script: |+
+  import yaml
```
[the remark this answers is not in the document]
Comment (anara123) [reply Scripts/script-DocumentationAutomation.yml]: We have yaml package in requirements.txt, so it should be available in demisto/python

**Comment by anara123** [comment]
@mayairv please review the english

**Comment by anara123** [comment]
image
I think there is a problem in the sentence

**Comment by anara123** [comment]
Also in my opinion it will look nicer if it will be html (with headers, bold/italic, etc) and not just plain text.
Wdyt @aviadl @Spivaka @dorsha ?

**Comment by aviadl** [comment]
It doesn’t really matter, since this only the data, then maya needs to edit it etc ...

Sent from my iPhone

On Dec 4, 2017, at 7:22 PM, Anar Azadaliyev <notifications@github.com<mailto:notifications@github.com>> wrote:


Also in my opinion it will look nicer if it will be html (with headers, bold/italic, etc) and not just plain text.
Wdyt @aviadl<https://github.com/aviadl> @Spivaka<https://github.com/spivaka> @dorsha<https://github.com/dorsha> ?

**Comment by mayairv** [comment]
Small fix:
"Click ‘Add instance’ to create and configure a new integration. You should configure the following settings:"

Also needed are Fetched incidents data, Top use-cases, Additional information and Known Limitations

**Pull request merged**